docs/faq.rst

```diff
@@ -65,27 +65,8 @@ I get import errors on libraries that depend on C modules
 
 This happens because our build system doesn't have the dependencies for building your project. This happens with things like libevent and mysql, and other python things that depend on C libraries. We can't support installing random C binaries on our system, so there is another way to fix these imports.
 
-You can mock out the imports for these modules in your ``conf.py`` with the following snippet::
+You can use this feature included in autodoc to mock http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_mock_imports.
 
-    import sys
-    from unittest.mock import MagicMock
-
-    class Mock(MagicMock):
-        @classmethod
-        def __getattr__(cls, name):
-            return MagicMock()
-
-    MOCK_MODULES = ['pygtk', 'gtk', 'gobject', 'argparse', 'numpy', 'pandas']
-    sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
-
-Of course, replacing `MOCK_MODULES` with the modules that you want to mock out.
-
-.. Tip:: The library ``unittest.mock`` was introduced on python 3.3. On earlier versions install the ``mock`` library
-    from PyPI with (ie ``pip install mock``) and replace the above import::
-
-        from mock import Mock as MagicMock
-
-If such libraries are installed via ``setup.py``, you also will need to remove all the C-dependent libraries from your ``install_requires`` in the RTD environment.
```

Comment:
We should however keep this notice, as a lot of folks have mocked their dependencies in the past, but not performed this step, assuming mocking was enough.

Reply:
@agjohnson , can you suggest what notice I can create for the folks?
For eg. "There is a new feature "autodock_mock_imports" for mocking the dependencies, the earlier feature present was a complex one".
Please suggest!

Reply:
I think this can be something like: "If you are mocking your dependencies using the mock library and such... "

Reply:
We probably just need to communicate that if you mock dependencies, in whatever fashion, that you aren't installing those dependencies either. The original copy just needs to be updated to match the new context i think.